### PR TITLE
Bugy fixes

### PR DESCRIPTION
**Pull request by Bod9001**
more to work on

### Changelog:
CL: [New] footprints now leave Detective Clues
CL: [New] Shields now protect Your arms and head

CL: [Improvement] Mapped in mutation grabber on all scenes
CL: [Improvement] increase the round end time for the Blob
CL: [Improvement] increase the round end for default ending
CL: [Balance] improved the RNG for dinosaurs you should see all types now

CL: [Fix] Fixes security console not showing Name#
CL: [Fix] Fixes sec light Producing red light
CL: [Fix] Fixes banana spawning multtiple banana peels
CL: [Fix] Fixes aliens spawning in witth Humans Attached 
CL: [Fix] no longer seelectable playable race slimeball
CL: [Fix] Fix scenario where Cancel surgery would break surgery
CL: [Fix] Fixes chemistry not working on head

### CODE REVIEW

**Comment by MaxIsJoe** [comment]
the seclite having a faint red light is not a bug? you're not fixing what is being already intended.

File: UnityProject/Assets/Scripts/Items/Food/Edible.cs [hunk 1]
```diff
@@ -200,29 +206,34 @@ public ReagentMix GetMixForBite(PlayerScript feeder)
...
+						}
+
+						if (added == false)
+						{
+							//If stackable has leavings and they couldn't go in the same slot, they should be dropped
+							pickupable.UniversalObjectPhysics.DropAtAndInheritMomentum(
+								ToDropOn.GetComponent<UniversalObjectPhysics>());
+						}
 					}
+					_ = Inventory.ServerDespawn(gameObject);
```
Comment (MaxIsJoe) [inline UnityProject/Assets/Scripts/Items/Food/Edible.cs:235]: this looks like it breaks my own fix

File: UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs [hunk 2]
```diff
@@ -99,7 +99,7 @@ public SecurityRecord GenerateSecurityRecord(PlayerScript script, JobType jobTyp
 		{
 			SecurityRecord record = new SecurityRecord();
 
-			record.EntryName = script.playerName;
+			record.EntryName = script.characterSettings.Name;
```
Comment (MaxIsJoe) [inline UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs:102]: didn't we already push a similar fix? `git merge` moment?

**Comment by corp-0** [comment]
tfw this will conflict with my shiet

**Comment by corp-0** [comment]
> CL: [Fix] Fixes security console not showing Name#

this was fixed already and tested on staging

File: UnityProject/Assets/Scripts/Objects/Medical/Genetics/GeneticEggLogic.cs [hunk 3]
```diff
@@ -11,9 +11,7 @@ public class GeneticEggLogic : MonoBehaviour
 
 	public int ChosenHatchTime;
 
-	public List<GameObject> Mobs000 = new List<GameObject>();
-	public List<GameObject> Mobs100 = new List<GameObject>();
-	public List<GameObject> Mobs200 = new List<GameObject>();
+	public List<GameObject> ALL = new List<GameObject>();
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/Medical/Genetics/GeneticEggLogic.cs]: the hell is ``ALL``

File: UnityProject/Assets/Scripts/Items/Food/Edible.cs (hunk 1 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/Food/Edible.cs:235]: tthis only triggers the ServerDespawn when  if (currentBites <= 0)

File: UnityProject/Assets/Scripts/Objects/Medical/Genetics/GeneticEggLogic.cs (hunk 3 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/Objects/Medical/Genetics/GeneticEggLogic.cs]: all mobs

File: UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs (hunk 2 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/UI/Systems/Jobs/CrewManifestManager.cs:102]: no? idk  Though

**Comment by Bod9001** [comment]
"this was fixed already and tested on staging" was still doing when I did it on Local

**Comment by Bod9001** [comment]
"the seclite having a faint red light is not a bug? you're not fixing what is being already intended. " Why is it red?

File: UnityProject/Assets/Scripts/Items/Food/Edible.cs (hunk 1 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Food/Edible.cs:235]: ...which is the intended behavior?

**Comment by MaxIsJoe** [comment]
> "the seclite having a faint red light is not a bug? you're not fixing what is being already intended. " Why is it red?

Because why not

File: UnityProject/Assets/Scripts/Objects/Medical/Genetics/GeneticEggLogic.cs (hunk 3 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Objects/Medical/Genetics/GeneticEggLogic.cs]: maybe remember it to be more clear then?

File: UnityProject/Assets/Scripts/Items/Food/Edible.cs (hunk 1 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/Items/Food/Edible.cs:235]: yeah, so it's fine?

**Comment by Bod9001** [comment]
it ends up just looking a bit pink, not  particularly good

**Comment by MaxIsJoe** [comment]
idk, i like it

File: UnityProject/Assets/Scripts/Items/Food/Edible.cs (hunk 1 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Food/Edible.cs:235]: I had to re-read this twice to make sure I'm not tripping.
but there's nothing preventing this from despawning the item before currentBites reaches zero, so you're re-introducing a bug that I already fixed.

**Review by MaxIsJoe — APPROVED** [review]

**Pull request merged**